cmd/root.go

```diff
@@ -224,6 +165,81 @@ func validateOutputFormat(output string) (writer.Format, error) {
 	return writer.NewFormat(format), nil
 }
 
+// Helper function to get the k8s config object
+func GetK8Config(a *MethodK8s) (*rest.Config, error) {
+	if a.RootFlags.ServiceAccountConfig.ServiceAccount {
```

Comment:
I think breaking out this `GetK8sConfigForServiceAccount` function would be really helpful for readability. Go is just such a verbose language that everything we can do to have clear function calls is doubly important.